smart-contracts/tasks/utils/zksync-contracts.ts

```diff
@@ -0,0 +1,8 @@
+import { task } from 'hardhat/config'
+import { getZkSyncBridgeContracts } from '../../lib/zksync'
+
+task('zksync-contracts', 'Show addresses of Zksync bridge contracts')
+  .addParam('chain', 'The id of a zksync chain')
+  .setAction(async ({ chain }) => {
+    console.log(await getZkSyncBridgeContracts(chain))
```

Comment:
console.log?

Reply:
yes it is just to show contracts addresses